app/src/main/java/com/hedvig/app/feature/embark/passages/multiaction/MultiActionAdapter.kt

```diff
@@ -0,0 +1,66 @@
+package com.hedvig.app.feature.embark.passages.multiaction
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.hedvig.app.R
+import com.hedvig.app.databinding.ViewMultiActionComponentBinding
+import com.hedvig.app.util.GenericDiffUtilItemCallback
+import com.hedvig.app.util.extensions.inflate
+import com.hedvig.app.util.extensions.viewBinding
+
+class MultiActionAdapter(
```

Comment:
I'd still like to discuss the implementation of this adapter, perhaps we can do it on a call 🙂